src/pages/index.js

```diff
@@ -112,51 +118,223 @@ const modalPreviewCloseButton = document.querySelector("#modalPreviewClose");
 //   document.removeEventListener("click", handleOverlayClose);
 // }
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "23172f33-55e2-4e0e-a695-0bae3ab40106",
+});
+
+let cardSection;
+
+api.getInitialCards().then((cards) => {
+  cardSection = new Section({ items: cards }, renderCard, "#section");
+
+  function renderCard(cards) {
+    const cardElement = new Card(
+      cards,
+      "#card-template",
+      handleImageClick,
+      handleCardDeleteClick,
+      handleCardLike,
+      handleCardLikeRemove
+    );
+    console.log(cards);
+    cardSection.addItem(cardElement.getView());
+  }
+
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**